atest/resources/atest_resource.robot

```diff
@@ -74,6 +74,14 @@ Execute
     ...    timeout=5min    on_timeout=terminate
     [Return]    ${result}
 
+Execute On Existing Execution Environment
+    [Arguments]    ${executor}    ${options}    ${sources}    ${default options}=
+    @{arguments} =    Get Execution Arguments    ${options}    ${sources}    ${default options}
+    ${result} =    Run Process    @{executor}    @{arguments}
+    ...    stdout=${STDOUTFILE}    stderr=${STDERRFILE}    output_encoding=SYSTEM
+    ...    timeout=5min    on_timeout=terminate
+    [Return]    ${result}
+
```

Comment:
We need this keyword because of missing option(s) in between test execution and post-processing. Example of this kind of option is ```--starttime```. Thus we need to execute test without processing the output. Then we need this new keyword as legacy ```Execute``` will set the execution environment causing the test output flushed. This is the rationale behind the new keyword. How generally this is needed? Probably not so much. I'll move it into the suite keywords.